sympy/polys/domains/realfield.py

```diff
@@ -61,10 +62,19 @@ def __hash__(self):
 
     def to_sympy(self, element):
         """Convert ``element`` to SymPy number. """
-        return Float(element, self.dps)
+        _mpf = element._mpf_
+        dps = prec_to_dps(_mpf[-1])
+        return Float(_mpf, dps)
```

Comment:
I'm not sure it's a good idea to try and preserve this information in elements of RR. The RealField implementation needs to be improved but it's supposed to represent a fixed precision domain in which all elements have the same precision.

Reply:
marking as resolved because I don't see a requirement that all be the same precision